spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/TraceKeys.java

```diff
@@ -459,6 +459,19 @@ public void setMethodNameKey(String methodNameKey) {
 	public static class Mvc {
 
 		/**
+		 * The timestamp used to provide a concrete moment in time when request processing
```

Comment:
looks good but curious why this would happen multiple times. if it is happening multiple times, it may be confusing (at least it is confusing to me..)